src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -679,13 +681,14 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
             item: (
                 <MenuItemWithTopDescription
                     key={translate('common.distance')}
-                    shouldShowRightIcon={!isReadOnly && isTypeRequest}
-                    title={iouMerchant}
+                    shouldShowRightIcon={!isReadOnly && canEditDistance}
+                    title={formattedDistance}
```

Comment:
I see that we're replacing the merchant with the distance only. Until we also add the rate field, let's continue to show the merchant which contains the distance and rate. Otherwise we wouldn't see the rate. Let's use getDistanceForDisplay within the getDistanceMerchant function, for now.